### PR TITLE
Fix release script and packaged app errors

### PR DESCRIPTION
**Pull request by ewc340**

### CODE REVIEW

**Comment by ewc340** [comment]
Still need to test this a bit more so won't merge this yet

**Review by HaileyJang — COMMENTED** [review]
good

**Review by HaileyJang — APPROVED** [review]
good

File: package.json [hunk 1]
```diff
@@ -105,10 +111,7 @@
     "ace-builds": "^1.4.12",
     "bufferutil": "4.0.1",
     "create-react-class": "15.6.2",
-    "electron": ">=7.2.4",
     "electron-json-storage": "4.0.2",
-    "eslint": "^7.5.0",
-    "eslint-plugin-jsdoc": "^30.2.1",
```
Comment (ewc340) [inline package.json:111]: These are basically devDependencies and don't need to be dependencies for the actual app, this way we save space for the size of the packaged app.

**Comment by ewc340** [comment]
This PR should be good to go after we fix the UID display issue. After that we should also finally merge to master.

File: package.json [hunk 2]
```diff
@@ -92,8 +96,10 @@
     "fs-extra": "5.0.0",
     "jszip": "3.1.5",
     "minimist": "1.2.3",
+    "mkdirp": "^1.0.4",
     "mocha": "5.0.0",
     "prettier": "^2.0.5",
+    "rimraf": "^3.0.2",
```
Comment (ewc340) [inline package.json:102]: These dependencies allow commands `mkdir -p` and `rm -rf` to be cross-platform (Windows, Mac, Linux).

**Review by AVSurfer123 — APPROVED** [review]

**Pull request merged**